lib/configs/recommended.js

```diff
@@ -55,6 +55,12 @@ const baseRecommendedConfig = {
             },
         ],
         'no-unused-expressions': ['error', { allowTernary: true }],
+        'padding-line-between-statements': [
+            'error',
+            { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
+            { blankLine: 'any', prev: ['const', 'let', 'var'], next: ['const', 'let', 'var'] },
```

Comment:
does that mean we can have lots of `consts` underneath each other, without needing blank lines` If so then 👍

Reply:
Yes - consts can be in one large block.